README.md

```diff
@@ -2,7 +2,15 @@
     <img alt="EpikCord logo" src="docs/EpikCord_Logo.png" width="200"> <!-- Yes I sourced this from DiscordGO because their design inspired me :D-->
 </p>
 
-# EpikCord.py
+
+![Documentation Status](https://readthedocs.org/projects/epikcordpy/badge/?version=latest&style=for-the-badge)
+![PyPi version](https://img.shields.io/badge/pypi-0.4.11-blue?style=for-the-badge)
```

Comment:
I don't want to change this each release.

Reply:
Well 1 sec

Reply:
i turned it off